lib/jxl/patch_dictionary_test.cc

```diff
@@ -23,47 +23,42 @@ namespace jxl {
 namespace {
 
 using ::jxl::test::ButteraugliDistance;
+using ::jxl::test::GetImage;
 using ::jxl::test::ReadTestData;
 using ::jxl::test::Roundtrip;
 
 TEST(PatchDictionaryTest, GrayscaleModular) {
-  JxlMemoryManager* memory_manager = jxl::test::MemoryManager();
   const std::vector<uint8_t> orig = ReadTestData("jxl/grayscale_patches.png");
-  CodecInOut io{memory_manager};
-  ASSERT_TRUE(SetFromBytes(Bytes(orig), &io));
+  extras::PackedPixelFile ppf;
+  ASSERT_TRUE(DecodeBytes(Bytes(orig), jxl::extras::ColorHints(), &ppf));
 
-  CompressParams cparams;
-  cparams.SetLossless();
-  cparams.patches = jxl::Override::kOn;
+  extras::JXLCompressParams cparams = jxl::test::CompressParamsForLossless();
+  cparams.AddOption(JXL_ENC_FRAME_SETTING_PATCHES, 1);
 
-  CodecInOut io2{memory_manager};
+  extras::PackedPixelFile ppf2;
   // Without patches: ~25k
-  size_t compressed_size;
-  JXL_EXPECT_OK(Roundtrip(&io, cparams, {}, &io2, _, &compressed_size));
+  size_t compressed_size = Roundtrip(ppf, cparams, {}, nullptr, &ppf2);
+  JXL_EXPECT_OK(&compressed_size);
```

Comment:
you probably meant 
JXL_EXPECT_OK(compressed_size);

Reply:
yes